### PR TITLE
Add instructions for Apache

### PR DESCRIPTION
**Pull request by Superdanby**
Apache needs the php package to work correctly.

### CODE REVIEW

File: start/sw/web-app/laravel5.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ OR, you can rip the benefit of using Fedora, and install composer directly inste
 $ sudo dnf install composer
 ```
 
+[Apache](/start/sw/web-app/apache.html) also needs the `php` package to work correctly.
+```bash
+# If you are using Apache as your http server:
+$ sudo dnf install php
```
Comment (pvalena) [inline start/sw/web-app/laravel5.md]: Please drop the PHP install line, as php installation is already mentined above.

**Comment by Superdanby** [comment]
I changed the instructions. Not sure if they're better now.

File: start/sw/web-app/laravel5.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ OR, you can rip the benefit of using Fedora, and install composer directly inste
 $ sudo dnf install composer
 ```
 
+You could also choose [Apache](/start/sw/web-app/apache.html) as the web server. But it requires an additonal package, `php`, to serve PHP contents properly. After installing the package, Apache needs to be restarted to take effect.
```
Comment (pvalena) [inline start/sw/web-app/laravel5.md]: This fits better on line 18 (`#### Install Nginx Web Server`) of this file.

Comment (pvalena) [reply start/sw/web-app/laravel5.md]: Also I think all of the web servers need to be reinstalled to be able to interpret php after it's installation.

Comment (Superdanby) [reply start/sw/web-app/laravel5.md]: I'll move it to line 18.
And I'm sure Apache doesn't need to be reinstalled after `php` installation.

Comment (pvalena) [reply start/sw/web-app/laravel5.md]: Sorry, I meant restarted.

File: start/sw/web-app/laravel5.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ This guide will help you setup your local development environment for Laravel, a
 #### Install Nginx Web Server
 Laravel Valet and Laravel recommends using Nginx. Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
 
+You can also choose [Apache](/start/sw/web-app/apache.html) as the web server. But it requires an additonal package, `php`, to serve PHP contents properly. After installing the package, Apache needs to be restarted to take effect.
```
Comment (pvalena) [inline start/sw/web-app/laravel5.md]: I still think the `php` related content is duplicated here, as the paragraph `#### Setup environment for PHP development` already comments on that. Also the `restart` sentence should probably be moved under the same PHP paragraph.

**Comment by pvalena** [comment]
Thanks!

**Pull request merged**